src/react/src/features/collection/Collection.tsx

```diff
@@ -0,0 +1,48 @@
+
+import { useAppSelector } from '../../app/hooks';
+import { Box, Button, Card, CardActions, CardContent, CardMedia, Chip, Grid, IconButton, Typography } from "@mui/material";
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Link } from 'react-router-dom';
+
+export function Collection() {
+  const collection = useAppSelector((state) => state.collection.collection);
+
+  return (
+    <Grid container spacing={2} justifyContent="center" >
+      {collection.map((artwork, index) => {
+        return (
+          <Grid item xs={11}>
+            <Card key={index} variant="outlined" style={{ paddingBottom: 2 }}>
+              <CardMedia
+                component="img"
+                sx={{ width: 200 }}
```

Comment:
what is this prop? I've never seen it

Reply:
the sx? It's a new way of styling in mui
https://mui.com/system/the-sx-prop/
https://stackoverflow.com/questions/52124938/how-to-add-padding-and-margin-to-all-material-ui-components